pegasus/sites.v3/code.org/public/accessibility.haml

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Accessibility'
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/accessibility-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.hero-banner-basic--accessibility
+  .wrapper
+    .flex-container
+      .text-wrapper.col-50.flex-container.align-items-center
+        .hero-banner-basic__content
+          %h1{style: "color: white"}
+            =hoc_s(:accessibility_hero_heading)
+          %p{style: "color: white"}
+            =hoc_s(:accessibility_hero_desc)
+      .col-50.flex-container.justify-end
+        %img{src: "/images/accessibility/hero-image.png", alt: ""}
+    .clear
+
+%section.accessibility-efforts.bg-neutral-light
+  .wrapper
+    %h2
+      = hoc_s(:accessibility_efforts_title)
+    %p
+      = hoc_s(:accessibility_efforts_desc)
```

Comment:
It looks like the `p.body-two` class is being applied to this paragraph, but it should be the default 16px/1rem paragraph size.

<img width="1052" alt="Screenshot 2023-07-18 at 2 29 11 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/13662bff-215e-4ae3-8696-5c17bd4c7b1c">

Reply:
done

Reply:
It's only that first paragraph that needed to be updated — the others in the blocks can remain 14px/0.875rem.